linker_scripts/extra.ld

```diff
@@ -0,0 +1,14 @@
+#include "boot.h"
+
+#if PLATFORM_IQUE
+
+_bootSegmentBssStart_CN_HI = _bootSegmentBssStart >> 16;
+
+_bootSegmentBssSize_CN_HI = _bootSegmentBssSize >> 16;
+
+bootproc_CN_HI = bootproc >> 16;
+
+BOOT_STACK_TOP_CN_HI = (sBootThreadStack + BOOT_STACK_SIZE) >> 16;
+BOOT_STACK_TOP       = (sBootThreadStack + BOOT_STACK_SIZE) & 0xFFFF;
```

Comment:
should we just define these all the time (like with no _CN suffix)? I feel that would be a bit simpler